stdlib/file/src/main/ballerina/src/file/file.bal

```diff
@@ -63,10 +72,13 @@ function externCreateDir(handle dir, boolean parentDirs) returns handle|Error =
 
 # Removes the specified file or directory.
 # If recursive flag is true, Removes the path and any children it contains.
+# ```ballerina
+# file:Error? results = file:remove("foo/bar.txt");
+# ```
 #
-# + path - String value of file/directory path.
-# + recursive - Indicates whether the `remove` should recursively remove all the file inside the given directory.
-# + return - Returns an `Error` if failed to remove.
+# + path - String value of file/directory path
+# + recursive - Indicates whether the `remove` should recursively remove all the file inside the given directory
+# + return - An `Error` if failed to remove
```

Comment:
```suggestion
# + return - An `Error` if failed to remove
```
Do we need to specify the error?

Reply:
The encompassing Error object is named as `Error`, I've added the module name to be more clear. Added in dba6e94